mathics/core/atoms.py

```diff
@@ -10,10 +10,16 @@
 import sympy
 from sympy.core import numbers as sympy_numbers
 
+try:  # pragma: no cover - optional dependency handled at runtime
```

Comment:
numpy is a dependence in sympy, so I guess it is safe to assume it is available.